tests/playwright/src/bootc-extension.spec.ts

```diff
@@ -61,6 +60,10 @@ beforeAll(async () => {
   const welcomePage = new WelcomePage(page);
   await welcomePage.handleWelcomePage(true);
   navBar = new NavigationBar(page);
+  const dashboardPage = await navBar.openDashboard();
+  await playExpect(dashboardPage.heading).toBeVisible();
+  await playExpect(dashboardPage.podmanStatusLabel).toBeVisible({ timeout: 30000 });
```

Comment:
I would lower the timeout here, so in total we do not exceed the basic hook timeout. also, podman extension should get up quite fast (<10s)

Reply:
lowered to 10s